threadit-web/src/pages/PostSpool/PostSpool.tsx

```diff
@@ -9,15 +10,14 @@ export default function PostSpool() {
     const profile = userStore.userProfile;
     const [title, setTitle] = React.useState('');
     const [lockInputs, setLockInputs] = React.useState(false);
-    const [interests, setInterests] = React.useState('');
-
-
+    const [tags, setTags] = React.useState(["General"]);
 
     const postSpool = async () => {
         if (profile) {
             setLockInputs(true);
             try {
-                await SpoolAPI.PostSpool(title, profile.id, interests.split(","), []);
```

Comment:
Can you pass interests here instead of joining and then immediately splitting on a comma?

Reply:
ya I hadnt realized that I was doing this lol